pkg/operator/ceph/cr_manager.go

```diff
@@ -156,11 +158,17 @@ func (o *Operator) startCRDManager(context context.Context, mgrErrorCh chan erro
 
 	// Set up a manager
 	mgrOpts := manager.Options{
-		LeaderElection:     false,
-		Namespace:          o.config.NamespaceToWatch,
-		MetricsBindAddress: "0",
-		Scheme:             scheme,
-		CertDir:            certDir,
+		LeaderElection: false,
+		Cache: cache.Options{
+			DefaultNamespaces: map[string]cache.Config{o.config.NamespaceToWatch: {}},
```

Comment:
I wonder if something like this might help? Seems unlikely if linter isn't complaining.

```suggestion
			DefaultNamespaces: map[string]cache.Config{o.config.NamespaceToWatch: cache.Config{}},
```